troubleshooting/known_issues/Gigabye_Missing_Redfish_Data.md

```diff
@@ -0,0 +1,18 @@
+### Gigabyte BMC Missing Redfish Data

```

Comment:
Need to add a statement about making sure the firmware is at the latest support firmware.

Reply:
Updated